src/views/Dog/Dog.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { useState, useEffect } from 'react';
 import { fetchDogsById } from '../../services/dog';
 import DogDetails from '../../components/DogDetails-v2';
+import { deleteDog } from '../../services/dog';
 
 export default function Dog(props) {
   const [dog, setDog] = useState([]);
```

Comment:
Dog is an object. It would make more sense to have the default state here be `{}` instead of `[]`